docs/conf.py

```diff
@@ -171,3 +193,7 @@
 
 # A list of files that should not be packed into the epub file.
 epub_exclude_files = ['search.html']
+
+from docutils.parsers.rst import directives
+from sphinx.directives.code import CodeBlock
+directives.register_directive('code', CodeBlock)
```

Comment:
Was this required for the ``` blocks in the openapi.yaml file?
If yes, is it possible to change the syntax highlighting language? Some code blocks look weird, for example https://demozxc.readthedocs.io/en/latest/api/#operation/get/series/instances (`605` is colored differently than the rest of the ID).

Reply:
Ah! I forgot to remove these lines. These were just to check something. 
Yes, I have changed the syntax highlighting language to `bash`